src/llmq/commitment.h

```diff
@@ -112,7 +114,7 @@ class CFinalCommitment
         obj.pushKV("signers", BitsVectorToHexStr(signers));
         obj.pushKV("validMembersCount", CountValidMembers());
         obj.pushKV("validMembers", BitsVectorToHexStr(validMembers));
-        obj.pushKV("quorumPublicKey", quorumPublicKey.ToString());
+        obj.pushKV("quorumPublicKey", quorumPublicKey.ToString(nVersion == LEGACY_BLS_NON_INDEXED_QUORUM_VERSION || nVersion == LEGACY_BLS_INDEXED_QUORUM_VERSION));
         obj.pushKV("quorumVvecHash", quorumVvecHash.ToString());
         obj.pushKV("quorumSig", quorumSig.ToString());
         obj.pushKV("membersSig", membersSig.ToString());
```

Comment:
looks like missing nVersion == ....

Reply:
I think better to remove whole function ToString() without arguments from CBLSWrapper to prevent misunderstandings like that:

```
diff --git a/src/bls/bls.h b/src/bls/bls.h
index 2cbe356d38..493d953b72 100644
--- a/src/bls/bls.h
+++ b/src/bls/bls.h
@@ -228,11 +228,6 @@ public:
         std::vector<uint8_t> buf = ToByteVector(specificLegacyScheme);
         return HexStr(buf);
     }
-
-    inline std::string ToString() const
-    {
-        return ToString(bls::bls_legacy_scheme.load());
-    }
 };
```

(in assumption that it is misusage)